### PR TITLE
Fixed broken link on profile comments

### PR DESCRIPTION
**Pull request by Kalekdan**

### CODE REVIEW

File: lib/forum-interactions.js [hunk 1]
```diff
@@ -61,15 +61,15 @@ export const linkCommentToQuestion = (questionID, commentID, commentList) => {
   }
   return new Promise((resolve, reject) => {
     axios
-      .put(`${API_URL}/questions/${questionID}`, {
```
Comment (andrewsrichardson) [inline lib/forum-interactions.js]: API interactions still need to point to questions I think. this was what I meant when I said it hadnt changed in strapi. comments should work without this change right?

**Review by andrewsrichardson — CHANGES_REQUESTED** [review]
was it defo broken before? this was what I meant by it being renamed but not actually changing the api in strapi. so page redirects should point to /community

File: lib/forum-interactions.js (hunk 1 above)
Comment (Kalekdan) [reply lib/forum-interactions.js]: That fix might not be required, but the comments links definitely are broken - if you go on someones profile and click the link to the post from the comments tab, it 404s

Comment (andrewsrichardson) [reply lib/forum-interactions.js]: yep. We can keep the links and revert the API call changes

**Review by andrewsrichardson — CHANGES_REQUESTED** [review]

**Pull request merged**